packages/react-core/src/components/Pagination/Pagination.tsx

```diff
@@ -210,8 +210,8 @@ export const Pagination: React.FunctionComponent<PaginationProps> = ({
     ofWord: 'of'
   },
   firstPage = 1,
-  page: pageProp = 0,
-  offset = 0,
+  page: pageProp = 1,
+  offset = null,
```

Comment:
could this be undefined?

Reply:
Other props in Pagination also have a default value of null like `itemsStart = null,  itemsEnd = null,`
and usually its not recomended to set values to `undefined`